.env.example

```diff
@@ -1,9 +1,7 @@
 SECRET_KEY=b0jbk58)l&^hy5cht+j$n9!laoke5ivdc+3&@qf2yrt1udvb85
 DEBUG=True
-DATABASE_URL=sqlite:///db.sqlite3
+DATABASE_URL=sqlite:///data/db.sqlite3
```

Comment:
what is the purpose of this?

Reply:
With compression enabled, static files get collected to STATIC_ROOT, which is why we need a sensible default for the development environment. If we later introduce media files, these will also need to be saved somewhere in the development setup. I propose to put these two and the database in a `data` folder by default.

When doing a proper installation, people can set a different DATA_DIR, or even customize static and media roots for serving via a webserver.

Reply:
DATA_DIR exists as a shortcut and as a way to create the directory for initial migration. sqlite failed on me when `data` did not exist to save `db.sqlite3` there. 

To answer your other comment, we could remove the default for DATA_DIR, but if you set custom DB, static and media files location, you don't need the DATA_DIR.

Reply:
i understand the point for the data dir.
But I'd still prefer having the STATIC_URL and STATIC_ROOT in the .env because this is really a thing that every installation should set. you can still default this to the data dir there?